// The Firmament project
// Copyright (c) 2011-2012 Malte Schwarzkopf <malte.schwarzkopf@cl.cam.ac.uk>
//
// TaskStateMessage indicates a state change of a task.

syntax = "proto3";

package firmament;

import "base/task_desc.proto";
import "base/task_final_report.proto";

message TaskStateMessage {
  uint64 id = 1;
  TaskDescriptor.TaskState new_state = 2;
  TaskFinalReport report = 3;
}
